push-notifications.js

```diff
@@ -34,19 +34,19 @@ function PushNotifications(options) {
     }
 }
 
-PushNotifications.prototype.publish = function(publishRequest) {
-    if (publishRequest === undefined) {
-        throw new Error('publishRequest argument is required');
-    }
-    if (!publishRequest.hasOwnProperty('interests')) {
-        throw new Error('"interests" is required in publishRequest');
+PushNotifications.prototype.publish = function(interests, publishRequest) {
```

Comment:
It might be useful to validate that interests is an array and that publishRequest is an object.